### PR TITLE
Setup SendGrid and test sending a single email.

### PR DESCRIPTION
**Pull request by l3x**
I added a versioned API controller.
```
├── controllers
│   ├── api
│   │   └── v1

```
I used jbuilder to render the json view
```
└── views
    ├── api
    │   └── v1
    │       └── sg_emails
```
And the Rspec's:
```
├── spec
│   ├── api
│   │   └── v1
│   │       ├── fixtures
│   │       │   └── sg_emails
│   │       ├── requests
│   │       └── spec
```

I thought I had a useful test framework.  It works.  It was interesting to work with and is arguably better than webmocks and VCR, but I hate it and am looking forward to replacing it.  

Oh, and the SendGrid email feature works for me :-)

In case you''re interested here's my .env file:

```
# Source this file prior to running rails server. Exmaple: . .env
export PGUSER=$USER
if [ "${PGPASSWORD}" == "" ]; then
    printf "Enter ${PGUSER}'s password: "
fi
export PGPASSWORD="${PGPASSWORD:-$(read -s pwd;echo $pwd)}"
export SENDGRID_API_KEY='<put_the_key_here>'
export APP_NAME='uhura'
export API_VER_NO="$(cat "lib/$(basename ${APP_NAME})/version.rb" | grep VERSION | head -n 1 | awk '{print $3}' | tr -d "'" | cut -d '.' -f1)"
export API_VER="api/v${API_VER_NO}"
export BASE_URI='http://localhost:3000'
export API_ENDPOINT="${BASE_URI}/${API_VER}/"
```

I load it by sourcing it:  `. .env`

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class UsersController < Admin::ApplicationController
     #
     # def index
     #   super
-    #   @resources = User.
+    #   @sendgrids = User.
```
Comment (knewter) [inline app/controllers/admin/users_controller.rb]: revert this change please

Comment (l3x) [reply app/controllers/admin/users_controller.rb]: done.
